apps/chat/src/components/Chat/Publish/HandlePublication.tsx

```diff
@@ -181,12 +181,14 @@ export function HandlePublication({ publication }: Props) {
       sectionName: t('Conversations'),
       dataQa: 'conversations-to-approve',
       Component: ConversationPublicationResources,
+      showTooltip: true,
     },
     {
       featureType: FeatureType.Prompt,
       sectionName: t('Prompts'),
       dataQa: 'prompts-to-approve',
       Component: PromptPublicationResources,
+      showTooltip: true,
```

Comment:
Why we don't want to show tooltip for file resources?

Reply:
refactored